src/apps/schema/src/appRevamp/components/FieldList.tsx

```diff
@@ -27,16 +27,19 @@ export const FieldList = () => {
   const { id } = params;
   const [search, setSearch] = useState("");
   const { data: models } = useGetContentModelsQuery();
-  const { data: fields, isLoading: isFieldsLoading } =
-    useGetContentModelFieldsQuery(id);
+  const {
+    data: fields,
+    isLoading: isFieldsLoading,
+    isFetching: isFieldsFetching,
+  } = useGetContentModelFieldsQuery(id);
   const [bulkUpdateContentModelField, { isLoading: isBulkFieldsUpdating }] =
     useBulkUpdateContentModelFieldMutation();
   const [draggedIndex, setDraggedIndex] = useState<number | null>(null);
   const [hoveredIndex, setHoveredIndex] = useState<number | null>(null);
   const showSpinner = isFieldsLoading;
   const model = models?.find((model) => model.ZUID === id);
 
-  useEffect(() => {
+  useMemo(() => {
```

Comment:
This is in relation to the change I've made on the `sortedFields` variable. I've noticed some weird UI behavior when I added the `isFieldsFetching` dependency. It causes `sortedFields` to be recalculated and briefly resets the field sort since it still has values for setDraggedIndex and setHoveredIndex. Changing this to useMemo makes sure that they get reset to null first so when sortedFields get recalculated it no longer shows the incorrect field sort.